pages/common/git-grep.md

```diff
@@ -0,0 +1,19 @@
+# git-grep
+
+> Find strings inside files as they existed anywhere in repo history. Accepts a lot of the same flags as regular grep. Learn more here: https://git-scm.com/docs/git-grep.
+
+- Search for a string in working directory:
```

Comment:
Perhaps this will read better?
```suggestion
- Search for a string in the current working directory:
```

Reply:
There is a slight distinction here that I didn't capture well, though I'm not sure its appropriate to describe in full in tldr. The command works on files that aren't ignored if its run against HEAD, otherwise it only works on files that are actually in that commit. The man page calls it the "work tree" in the former case.

How about if we changed it to "Search for a string in non-ignored files in the current directory:"?

Reply:
We also need to mention subdirectories. Let us use the word "tracked" rather than "non-ignored". Better to use a positive than double-negative. How about - "Search for a string in all tracked files in the working directory and its subdirectories" ?